docs/javascript/v2/features/hls.mdx

```diff
@@ -173,6 +173,79 @@ function HLSView({ hlsUrl }) {
 
 > **Note:** If you wish to customize the stream, please check [this guide](./../foundation/live-streaming#step-4-customize-the-stream) for more information.
 
+## Changing the quality of Stream(Hls.js)
+
+You may want to change the quality layer of the stream based on user input or some other criteria. Each library handles this differently.
+Below we mention an example snippet on how to do it if you choose to use hls.js. Please refer to the respective documentations of the library
+you use.
+
+<Tabs id="hlsChangeQuality" items={['JavaScript', 'React']} />{' '}
+
+<Tab id='hlsChangeQuality-0'>
+
+```js
+import Hls from 'hls.js';
+
+function changeQuality() {
+    
+    if (Hls.isSupported()) {
+        let hls = new Hls();
+        console.log("Available quality levels in current stream", hls.levels);
+        setCurrentLevel(hls.levels[0]); //set the first level.
+    }
+    
+}
+
+  function setCurrentLevel(currentLevel) {
+    // make sure we are choosing the right level by checking the
+    // height and width.
+    const newLevel = this.hls.levels.findIndex(
+      level =>
+        level.height === currentLevel.height &&
+        level.width === currentLevel.width
+    );
+    // set it.
+    this.hls.currentLevel = newLevel;
+  }
+```
+
+</Tab>
+
+<Tab  id='hlsChangeQuality-1'>
+
+```jsx
+import Hls from 'hls.js';
+
+function HLSView({ hlsUrl }) {
+    const videoRef = useRef(null);
+    useEffect(() => {
+        if (videoRef.current && hlsUrl) {
+            if (Hls.isSupported()) {
+                let hls = new Hls();
+                console.log("Available quality levels in current stream", hls.levels);
+                setCurrentLevel(hls.levels[0]); //set the first level.
+            }
+        }
+    }, [hlsUrl]);
+
+    function setCurrentLevel(currentLevel) {
```

Comment:
format properly. seems off

Reply:
formatted the entire document with eslint now. let me know if something else is needed.